schemas/remote_configuration_v2_template.schema.json

```
@@ -0,0 +1,41 @@
+{
+  "$id": "remote_configuration_v2_template",
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "additionalProperties": false,
+  "description": "Config template files stored as object which contain list of conditional gathering rules and list of container logs",
```

If I didn't know what the data look like, I would probably understand that this is a schema of an object that stores the content of files, each file containing a list of conditional gathering rules and list of container log requests.

How about this?

```suggestion
  "description": "Template of a remote configuration for the Rapid Recommendations feature of the Insights Operator.",
```